dotcom-rendering/src/components/StickyBottomBanner/ReaderRevenueBanner.tsx

```diff
@@ -315,20 +301,10 @@ const RemoteBanner = ({
 			});
 	}, [module]);
 
-	useOnce(() => {
-		const { componentType } = meta;
-
-		// track banner view event in Google Analytics for subscriptions banner
-		if (componentType === componentTypeName) {
-			trackNonClickInteraction(displayEvent);
-		}
```

Comment:
Worth checking with the team responsible for banners if they would like this to be tracked somewhere, still…

Reply:
I understand that all teams are aware that GA will be removed. Do you know someone in that team that we could tag in here?

Reply:
Hey @tomrf1, could you please let us know what do you think about this?

Reply:
Yep this is fine, we have separate tracking of component_events via ophan